src/components/mgt-people/mgt-people.ts

```diff
@@ -270,6 +293,8 @@ export class MgtPeople extends MgtTemplatedComponent {
           this.people = await getPeopleFromGroup(graph, this.groupId);
         } else if (this.userIds) {
           this.people = await getUsersForUserIds(graph, this.userIds);
+        } else if (this.peopleQuery) {
+          this.people = await getUsersForUserIds(graph, this.peopleQuery);
```

Comment:
This won't work for queries like "Megan Bowen" or email addresses from contacts. You should use a similar approach to how `personQuery` works in person.

Reply:
Didn't know that was supported. Will update!